examples/penumbra/penumbra_ibc_test.go

```diff
@@ -0,0 +1,660 @@
+package penumbra
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"cosmossdk.io/math"
+	transfertypes "github.com/cosmos/ibc-go/v8/modules/apps/transfer/types"
+	"github.com/strangelove-ventures/interchaintest/v8"
+	"github.com/strangelove-ventures/interchaintest/v8/chain/penumbra"
+	"github.com/strangelove-ventures/interchaintest/v8/ibc"
+	"github.com/strangelove-ventures/interchaintest/v8/relayer"
+	"github.com/strangelove-ventures/interchaintest/v8/testreporter"
+	"github.com/strangelove-ventures/interchaintest/v8/testutil"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+)
+
+// TestPenumbraToPenumbraIBC asserts that basic IBC functionality works between two Penumbra testnet networks.
+// Two instances of Penumbra will be spun up, the relayer will be configured, and an ics-20 token transfer will be
+// sent from chainA -> chainB successfully. At the end two more ics-20 transfers will be sent from chainA -> chainB,
+// these transfers will time out due to the timeout height and timeout timestamp being reached respectively.
+func TestPenumbraToPenumbraIBC(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping in short mode")
+	}
+
+	t.Parallel()
+	client, network := interchaintest.DockerSetup(t)
+
+	nv := 1
+	fn := 0
+
+	chains, err := interchaintest.NewBuiltinChainFactory(zaptest.NewLogger(t), []*interchaintest.ChainSpec{
+		{
+			Name: "penumbra",
+			// Version: "v0.77.2,v0.37.5",
+			Version: "local,v0.37.5",
```

Comment:
The version strings will need to be updated when a new Penumbra release lands

Reply:
is this meant to be v0.78.0? (released 2 hours ago) or another part of the 0.77.X line?

Reply:
Yes, this should be `v0.78.0`

Reply:
A Heighliner/Docker image release will need to be published for `v0.78.0` as well

Reply:
[Published](https://github.com/strangelove-ventures/heighliner/pkgs/container/heighliner%2Fpenumbra/235565547?tag=v0.78.0)!

Reply:
resolved in: https://github.com/strangelove-ventures/interchaintest/pull/1157/commits/41c2aaa688e88a827e06d0fe3a2297ab8694d28e